docs/api/data-views.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[data-views-api]]
```

Comment:
This changes docs URLs, correct? 

In this case we should add this to redirects: https://github.com/elastic/kibana/blob/main/docs/redirects.asciidoc

Reply:
addressed